server/app/src/data_harvester.py

```diff
@@ -121,48 +121,25 @@ def copy_snapshots(self, datasets: List[StandardizedDataset]):
         # Download the dataset locally
         tmp_folder: str = self.index.downloadToTmp(self.path)
 
-        # Deal with the original skeleton model
-        skeleton_preset = 'vicon'
-        if os.path.exists(tmp_folder + '_subject.json'):
-            subject_json = json.load(open(tmp_folder + '_subject.json'))
-            if 'skeletonPreset' in subject_json:
-                skeleton_preset = subject_json['skeletonPreset']
-                # Ensure that the _subject.json file is set to custom for the upload in the standardized folder
-                subject_json['skeletonPreset'] = 'custom'
-                json.dump(subject_json, open(tmp_folder + '_subject.json', 'w'))
+        # Identify trials that are too short. We don't want to process these
+        trials_to_remove = self.id_short_trials(tmp_folder)
```

Comment:
Just for clarity to the reader, now that we've factored out this method, (so they know these are strings that represent paths), I'd add a type signature and maybe make the variable name more explicit here:

`trial_paths_to_remove: List[str] = self.id_short_trials(tmp_folder)`